### PR TITLE
[bitnami/*] Update redis subchart to new major

### PR DESCRIPTION
**Pull request by FraPazGal**
**Description of the change**

Updates the redis subchart to its new major, 13.0.0. Bumps the affected charts major version and adds the related notice.

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Review by juan131 — CHANGES_REQUESTED** [review]
Block until https://github.com/bitnami/charts/pull/6102 is done

**Comment by FraPazGal** [comment]
Blocked until #6192 is merged.

**Review by juan131 — APPROVED** [review]
LGTM! Thanks so much @FraPazGal

**Pull request merged**
